docs/pages/en/4.providers/imagekit.md

```diff
@@ -50,7 +50,7 @@ This can be used to blur an image. Use modifier `blur` to specify the Gaussian B
 ```html
 <nuxt-img
   provider="imagekit"
-  src="/default-image.jpg"
+  src="/default-image.jpeg"
```

Comment:
Why was the extension changed to `.jpeg`?

Reply:
Because with `.jpg` the image was not loading for me

Reply:
That may be because your original default image was deleted, and another one was uploaded with the `.jpeg` extension.

However, the original default image that is uploaded has `.jpg` extension.
https://ik.imagekit.io/nndev/default-image.jpg

Reply:
Ok, I will make the change to `.jpg` in that case.